truffle/com.oracle.truffle.dsl.processor/src/com/oracle/truffle/dsl/processor/InstrumentRegistrationProcessor.java

```diff
@@ -57,24 +57,34 @@ public SourceVersion getSupportedSourceVersion() {
         return SourceVersion.latest();
     }
 
+    private static final int NUMBER_OF_PROPERTIES_PER_ENTRY = 4;
```

Comment:
Don't know. Using a named constant was the simplest solution I could think off to know how many instruments are in a file (line 117). Otherwise, I'd have to count id entries or something. Might be more robust, but it's also more code to maintain.